### DISCUSSION TITLE
Textual Tutorial

### DISCUSSION THREAD

**Discussion by Zengyf-CVer** [Textualize/textual#618] [Q&A]
Hello, I read all the content in the README.md, but I think it is not enough, because I want to develop a software, is there a more detailed tutorial about Textual?

**Comment by jeffwright13** [comment]
You could try [this](https://dev.to/wiseai/textual-the-definitive-guide-part-1-1i0p).

**Reply by Zengyf-CVer** [reply]
Thank you very much！
